### DISCUSSION TITLE
Resolution-dependent `ls_background_image`

### DISCUSSION THREAD

**Discussion by rathann** [neutrinolabs/xrdp#2026] [Ideas]
It would be nice to have an option to load a different login scree background image (`ls_background_image` `xrdp.ini` option) depending on client resolution.

**Comment by matt335672** [comment]
Hi @rathann,

The development version of xrdp includes image scaling and zooming support - see #1962 

Are you asking for something over and above what this provides?

**Comment by rathann** [comment]
My company provides a set of branded backgrounds tailored to different resolutions, eg. 1920x1080, 2560x1440, 3840x2160, etc. I'm looking for a way to select the appropriate image based on client resolution if possible.

**Comment by matt335672** [comment]
That's interesting - thanks.

I'm going to convert this to a a discussion so we can (hopefully) widen out the conversation a bit.

**Comment by matt335672** [comment]
One thing which occurred to me in thinking about this was that resolution is one way to think about selecting a different image, but there may be others (i.e.):-
1. Aspect ratio
2. Colour depth

This might be solved by a bit of scripting, but I'm a bit reticent to go down this route as xrdp runs with privilege on many system.

Comments welcome, particularly for real-world usages. Am I over-thinking this above?
